src/languages/fsharp.js

```diff
@@ -127,20 +123,47 @@ export default function(hljs) {
         ]
       },
       {
-        className: 'meta',
-        begin: '\\[<',
-        end: '>\\]',
-        relevance: 10
+        // computation expressions:
+        beginScope: "emphasis",
+        match: /\b[_a-z]\w*(?=\s*\{)/i
+      },
+      {
+        // preprocessor directives and fsi commands:
+        scope: 'meta',
+        begin: '#',
```

Comment:
Can this be anywhere on a line or only the beginning?

Reply:
This likely needs to be more precise as right now it'll match say comments in many other languages.  You may need to match the keywords with a lookahead to ensure it's truly a preprocessor and not a false positive.

Reply:
> Can this be anywhere on a line or only the beginning?

It has to be the first non-whitespace thing on a line.

I used the same matching used for `csharp` as I believe it's similar.

Would the following be better?
```js
  {
    // preprocessor directives and fsi commands:
    scope: 'meta',
    begin: /^\s*#(if|else|endif|line|nowarn|light|r|i|I|load|time|help|quit)/i,
    end: /\s|$/
  },
```

(I'm not sure if there is a better way to do a look-ahead without repeating all the keywords)

Reply:
Yes, but please put them all in a nice tidy array and then use our regex tools to build the regex expression.  And you might want a multi-matcher here because technically the `\s*` should not be part of the highlighted content. (we do try to be strict about our whitespace boundaries when it's easy to do so).

`end` should probably be a look-ahead here for the same reason, to not consume the space inside the scope.